biztalk/core/operational-data-service.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Operational Data Service | Microsoft Docs"
+description: Operational Data Service 
+ms.custom: "biztalk-2020"
+ms.date: "1/7/2020"
+ms.prod: "biztalk-server"
+ms.reviewer: ""
+
+ms.suite: ""
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+ms.assetid: fe6d3a97-c7c0-4147-baa9-ee12f93248eb
+caps.latest.revision: 11
+author: MandiOhlinger
+ms.author: mandia
+manager: "anneta"
+---
+# Operational Data Service
+
+**Starting with [!INCLUDE[bts2016_md](../includes/bts2016-md.md)] [!INCLUDE[featurepack1](../includes/featurepack1.md)]**, BizTalk Server provide the operational data service, which can be accessed or queried using OData visualization tools, like Power BI. [What is OData?](https://docs.microsoft.com/en-us/odata/overview)
+
+## What is operational data
+Operational data is information on the instances and messages flowing through your [!INCLUDE[btsBizTalkServerNoVersion_md](../includes/btsbiztalkservernoversion-md.md)] environment. The operational data feed is the same data you get looking at Group Hub in [!INCLUDE[btsBizTalkServerAdminConsoleui_md](../includes/btsbiztalkserveradminconsoleui-md.md)].
+
+The feed includes the following data tables:
+* Application data
+* AS2 Status Records
+* Batching information
+* Instance information
+* Interchange Aggregations Records
+* Interchange Status Records
+* Messages
+* Subscriptions
+* Tracked Events
+* Transaction reports
+* Transaction sets
+
+> [!TIP]
+> [PowerBI.com](https://powerbi.microsoft.com) is a great resource to understand and learn more about Power BI.
+
+## Prerequisites
+* Download and install [Power BI Desktop](https://powerbi.microsoft.com/desktop/) on any computer that has network access to your [!INCLUDE[btsBizTalkServerNoVersion_md](../includes/btsbiztalkservernoversion-md.md)]
+* Install IIS on the [!INCLUDE[btsBizTalkServerNoVersion_md](../includes/btsbiztalkservernoversion-md.md)]. 
+* [Configure the REST APIs](../install-and-config-guides/configure-biztalk-server.md) for **Operational data service**.
+* Optional. Install and configure a [Power BI Gateway](https://powerbi.microsoft.com/gateway/) to connect [PowerBI.com](https://powerbi.microsoft.com) with your on-premises BizTalk Server. If you're not using an on-premises BizTalk Server, then you don't need the gateway.
+
+
+## Use Power BI to Load Operational Data.
+
+1. Download and install the [Power BI Desktop](https://powerbi.microsoft.com/desktop/) on your BizTalk Server. You can select to open it, which is optional. If you have a work or school account, you may have access to Power BI. Try signing in with that account. Or, you can try it for free after signing up.
+2. Open the `\Program Files (x86)\Microsoft BizTalk Server 2016\OperationalDataService` folder, and open the `BizTalkOperationalData.pbit` file:
```

Comment:
Should we remove '2016' from the folder path in the text, and the screenshot?

Reply:
Yes!